web/blueprints/finance/__init__.py

```diff
@@ -896,6 +898,21 @@ def transaction_confirm(transaction_id):
     return redirect(url_for('.transactions_unconfirmed'))
 
 
+@bp.route('/transaction/confirm_selected', methods=['GET', 'POST'])
```

Comment:
Since you don't render a form here, there's no need for `GET`:
```suggestion
@bp.route("/transaction/confirm_selected", methods=["POST"])
```

Reply:
the tested failed when I removed that get and then I was like it is how it is probably cause all the outer functions used get an redirect to select the specified transaction

Reply:
Ah, I see, that's an oversight in the permissions test. It always uses `GET`… let me see whether I can fix that quickly.